ui/raidboss/data/07-dt/raid/r4s.ts

```diff
@@ -2001,6 +2302,21 @@ const triggerSet: TriggerSet<Data> = {
           cn: '黄激光 (${loc}) - 打向 ${bait}',
           ko: '노란 레이저 (${loc}) - ${bait}쪽으로',
         },
+        northTower: {
+          en: 'North Tower',
+        },
+        southTower: {
+          en: 'South Tower',
+        },
+        eastTower: {
+          en: 'East Tower',
+        },
+        westTower: {
+          en: 'West Tower',
+        },
+        specificCannon: {
+          en: '${loc} Cannon - Bait ${bait}',
+        },
```

Comment:
Wouldn't it be possible to use `yellowShort` and `blueShort` for this?

Reply:
i'll see. I originally had a different text string for both strategies, and I didn't find the tower color callout to be helpful but perhaps others might like it since they can confirm they have the right color of tower.

Reply:
And it would make the code simpler.